frontend/src/entry-forms/AddPostingMetadataButton.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
```

Comment:
This seems to be nearly an identical copy of the corresponding component for entries - please reuse that one